docs/developers/schema_versions.md

```diff
@@ -24,3 +26,15 @@ To aid with this process, the following guidelines are provided:
 In general, a schema version should be adjusted if and only if there's a change in the schema's definition. This is typically reserved for major releases _(e.g. upgrading from iRODS 4.2 to iRODS 4.3)_.
 
 A _change in the schema definition_ is defined as the addition, modification, and/or removal of a JSON property or database table/column.
+
+### iRODS 5 and later
+
+!!! Important
+    Starting with iRODS 5, version numbers consist of three segments, _X.Y.Z_, where _X_ is the major version number, _Y_ is the minor version number, and _Z_ is the patch version number. A major version change consists of the major version number being incremented. For example, upgrading from iRODS 4.3.4 to iRODS 5.0.0 or iRODS 5.1.0 to iRODS 6.0.0.
+
+The guidelines defined for iRODS 4 still apply to iRODS 5, but iRODS 5 improves upon the situation by reducing management of schema versions to two CMake variables, defined in the top-level CMakeLists.txt file. They are as follows:
```

Comment:
That's a long statement. Probably needs to be split into two statements.

Reply:
i don't think the !!! important tag is necessary.  it's in the new 5 and later section... two paragraphs seem fine as is, just without being in a special box?

Reply:
Done.